syntax = "proto3";

package mprpc;

message RpcHeader 
{
    bytes service_name = 1; // Service name
    bytes method_name = 2; // Method name
    uint32 arg_size = 3; // Size of the argument data
}